### ISSUE TITLE
Error in daemon_spotter.php

### ISSUE THREAD

**Issue by dterweij** [Ysurac/FlightAirMap#395]
```
DATA : hex : 780DB0 - ident : CSN345 - ICAO : A333 - Departure Airport : ZBAA - Arrival Airport : EHAM - Latitude : 53.15291 - Longitude : 6.94901 - waypoints :  - Altitude : 234 - Heading : 234 - Speed : 469 - Departure Airport Time :  - Arrival Airport time :  - Pilot :  - Source name : DUMP1090
\o/ Add CSN345 from sbs in Live DB : -- Delete previous data -- (Add to Spotter archive : Error : SQLSTATE[42000]: Syntax error or access violation: 1305 FUNCTION air.ST_Within does not exist
success)success
```

**Comment by Ysurac** [comment]
What is the DB used ? MariaDB, MySQL or PostgreSQL ? And what version ?
For MySQL you need at least version 5.6.1.
For PostgreSQL PostGIS must be installed.

**Comment by dterweij** [comment]
```
$ mysql --version
mysql  Ver 14.14 Distrib 5.5.59, for Linux (x86_64) using readline 5.1
$ cat /etc/centos-release
CentOS release 6.9 (Final)
```

**Comment by Ysurac** [comment]
MySQL 5.6.x is available since 2012, I can't support MySQL 5.5 because of the lack of spatial extensions  support.

You should be able to install it from the Official MySQL Community repository.

**Comment by dterweij** [comment]
I rather stay at stable stock packages. Newest not always safest way :)
